open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java

```diff
@@ -0,0 +1,266 @@
+import io.opensphere.mantle.plugin.selection.impl.CreateBufferCommand;
+import io.opensphere.mantle.plugin.selection.impl.CreateBufferSelectedSegmentCommand;
+import io.opensphere.mantle.plugin.selection.impl.DeselectCommand;
+import io.opensphere.mantle.plugin.selection.impl.LoadFeaturesCommand;
+import io.opensphere.mantle.plugin.selection.impl.LoadFeaturesCurrentFrameCommand;
+import io.opensphere.mantle.plugin.selection.impl.RemoveAllFeaturesCommand;
+import io.opensphere.mantle.plugin.selection.impl.SelectCommand;
+import io.opensphere.mantle.plugin.selection.impl.SelectExclusiveCommand;
+
+/**
+ *
+ */
+public class SelectionCommandFactory
+{
+    /** Logger reference. */
+    private static final Logger LOGGER = Logger.getLogger(SelectionCommandFactory.class);
+
+    /** The Constant FILTERS_ACTIVE. */
+    public static final String FILTERS_ACTIVE = " (Filters Active)";
+
+    /** ADD_FEATURES. */
+    public static final SelectionCommand ADD_FEATURES = new AddFeaturesCommand();
+
+    /** ADD_FEATURES_CURRENT_FRAME. */
+    public static final SelectionCommand ADD_FEATURES_CURRENT_FRAME = new AddFeaturesCurrentFrame();
+
+    /** The CANCEL_QUERY. */
+    public static final SelectionCommand CANCEL_QUERY = new CancelQueryCommand();
+
+    /** Buffer Line. */
+    public static final SelectionCommand CREATE_BUFFER_REGION = new CreateBufferCommand();
+
+    /** Buffer Line. */
+    public static final SelectionCommand CREATE_BUFFER_REGION_FOR_SELECTED_SEGMENT = new CreateBufferSelectedSegmentCommand();
+
+    /** DESELECT. */
+    public static final SelectionCommand DESELECT = new DeselectCommand();
+
+    /** The LOAD_FEATURES. */
+    public static final SelectionCommand LOAD_FEATURES = new LoadFeaturesCommand();
+
+    /** The LOAD_FEATURES_CURRENT_FRAMEe. */
+    public static final SelectionCommand LOAD_FEATURES_CURRENT_FRAME = new LoadFeaturesCurrentFrameCommand();
+
+    /** PURGE. */
+    public static final SelectionCommand REMOVE_ALL = new RemoveAllFeaturesCommand();
+
+    /** SELECT. */
+    public static final SelectionCommand SELECT = new SelectCommand();
+
+    /** SELECT_EXCLUSIVE. */
+    public static final SelectionCommand SELECT_EXCLUSIVE = new SelectExclusiveCommand();
+
+    /**
+     * Default set of commands.
+     */
+    private static final List<SelectionCommand> DEFAULT_COMMANDS = List.of(ADD_FEATURES, ADD_FEATURES_CURRENT_FRAME,
+            LOAD_FEATURES, LOAD_FEATURES_CURRENT_FRAME, CANCEL_QUERY, CREATE_BUFFER_REGION,
+            CREATE_BUFFER_REGION_FOR_SELECTED_SEGMENT, DESELECT, REMOVE_ALL, SELECT, SELECT_EXCLUSIVE);
+
+    /**
+     * Default set of commands.
+     */
+    private static final Map<String, SelectionCommand> DEFAULT_COMMAND_MAP = DEFAULT_COMMANDS.stream()
+            .collect(Collectors.toMap(c -> c.getName(), c -> c));
+//
+//    /**
+//     * Default set of commands.
+//     */
+//    private static final Map<String, SelectionCommand> CLASSNAME_COMMAND_MAP = DEFAULT_COMMANDS.stream()
```

Comment:
Do you want to leave these commented blocks as is?

Reply:
fixed